### DISCUSSION TITLE
Can i used isso for a store-like website?

### DISCUSSION THREAD

**Discussion by CosteaPaul** [posativ/isso#921] [Q&A] [answered]
Hi all,

I have an ancient store-like website, that hits a /product.cgi page. Each product has its id, so whenever you're looking at a product you're hitting something like /product.cgi?prod_id=234xxFr2.

I would like to add the ability to comment on each product, but skimming the documentation makes me think the comment section is specifically connected to the URL and thus this won't work out of the box. Is this a correct interpretation? :)  

Thanks for the support,
-Paul

**Comment by ix5** [comment, marked as answer]
By default, Isso uses the URL without any question-mark parameters (e.g. `https://foo.bar/baz?foo` will receive, through Javascript `window.location.pathname`, the internal ID `/baz`)

You can override the thread id for each page by setting `data-isso-id` in the embed part, see [docs: Client config](https://isso-comments.de/docs/reference/client-config/). That way, you can still have URL query-param-based routing.

Another option would be to change the embed script to use `window.location.href` instead.

**Comment by jelmer** [comment]
A side-note though is that you probably want something heavier (with a proper database backing it) if this is a high volume web site.
